corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -204,6 +204,10 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             'click @ui.searchForBlank': 'toggleBlankSearch',
         },
 
+        _render: function () {
+            _.delay(this.render);
```

Comment:
What does this accomplish - move it off thread?  Doesn't `_.delay` take a [`wait` arg](https://underscorejs.org/#delay)?

Reply:
Yeah, the intent was not to render until the current call stack is complete (including closing the select2). Looking at the underscore docs, I meant to use `defer`, which should be equivalent but more readable. Updated in https://github.com/dimagi/commcare-hq/pull/31958/commits/3bf2625526eb7e56efe5a9085a53d5285590d384